doc/changes/changes_6.1.0.md

```diff
@@ -0,0 +1,30 @@
+# r-exasol 6.1.0, released t.b.d.
```

Comment:
I am not sure, if we should stay at 6.1.0, this feels almost like a major version change

Reply:
Bumped it to 7.0.0